### ISSUE TITLE
Sensei Ten Support

### ISSUE THREAD

**Issue by HolyStephano** [libratbag/libratbag#860]
Not a problem, but the Sensei Ten from steelseries launched recently. Has the same basic layout as the other senseis (at least the raw) with RGB lighting included on the scrollwheel, and steelseries logo in the palm.

I'd hope it would be fairly simple to build support for it, but i personally am scared to attempt to just modify the old sensei files to get it working.

Currently i'm just using a VM with USB passthrough to modify the mouse settings.

**Comment by HolyStephano** [comment]
I'd be willing to do any debugging necessary, as i do have the device on-hand. I have an arch machine with a windows VM to test with.

**Comment by FFY00** [comment]
Yes, please modify the settings one by one in the VM and capture the packets with wireshark.

Please have one wireshark session log per action.

We can start with the DPI. Can you get me the following logs?

- Set DPI 1 to min
- Set DPI 1 to next step
- Set DPI 1 to next step
- Set DPI 1 to 1000
- Set DPI 1 to max
- Set DPI 2 to min
- Set DPI 2 to next step
- Set DPI 2 to max

Repeat the steps from DPI 2 for every other DPI setting, in case you have more than two DPI settings.

**Comment by HolyStephano** [comment]
I'm not too familiar with capturing with wireshark, let alone logging with it. Sorry if it's any hassle, but could you give me some more explicit directions?

**Comment by HolyStephano** [comment]
Issues i'm seeing with doing this already: Steelseries engine (At least with this mouse) allows up to 5 settings (I only personally use 1) and whenever i move the DPI/CPI slider past one of the other settings in either direction (either lower or higher DPI) it shifts where it is in the order.

**Comment by FFY00** [comment]
I am not sure I follow. Can you explain better or send a screenshots/video?

**Comment by HolyStephano** [comment]
Here's a video about that: https://f.cloudninja.pw/2019-12-14_23-35-20.mp4

**Comment by HolyStephano** [comment]
I have an actual windows install now that i can use, if that makes it easier.
